packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -297,26 +311,54 @@ class _RawVectorGraphicsWidget extends SingleChildRenderObjectWidget {
   const _RawVectorGraphicsWidget({
     Key? key,
     required this.pictureInfo,
+    required this.color,
+    required this.blendMode,
   }) : super(key: key);
 
   final PictureInfo pictureInfo;
+  final Color? color;
+  final BlendMode blendMode;
 
   @override
   RenderObject createRenderObject(BuildContext context) {
-    return _RenderVectorGraphics(pictureInfo);
+    return _RenderVectorGraphics(
+      pictureInfo,
+      color,
```

Comment:
It'd be nice to support color filter directly. Flutter_svgs impl is a remnant of when we only had mode filters.

Reply:
done